portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisable.testcase

```diff
@@ -524,4 +524,150 @@ definition {
+    When the 'Enable Segmentation' is disabled
+    Then segmentation is disabled for Collections"
+	@priority = "4"
+	test ViewDisabledSegmentationAlertAtDynamicCollectionEditor {
+		task ("Create a Collection via JSONWS") {
+			JSONAssetlist.addDynamicAssetListEntry(
+				groupName = "Test Site Name",
+				title = "Dynamic Collection Test");
+		}
+
+		task ("Navigate to Collection Admin") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
+		}
+
+		task ("Check if warning appears") {
+			Segmentation.checkSegmentationDisabledWarning();
+		}
+	}
+
+	@description = "LPS-151362: Given segmentation is disabled at Instance Settings
+	When user navigates to Dynamic Collection editor
+	Then the warning alert explains how to re-enable it"
+	@priority = "3"
+	test ViewReEnableAlertAtDynamicCollectionEditor {
+		task ("Create a Collection via JSONWS") {
+			JSONAssetlist.addDynamicAssetListEntry(
+				groupName = "Test Site Name",
+				title = "Dynamic Collection Test");
+		}
+
+		task ("Navigate to Collection Admin") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
+		}
+
+		task ("Check if warning appears") {
+			Segmentation.checkSegmentationReEnableWarning();
+		}
+	}
+
+	@description = "LPS-151362: Given segmentation is disabled at Instance Settings
+	When user navigates to Manual Collection editor
+	Then the warning alert explains how to re-enable it"
+	@priority = "3"
+	test ViewReEnableAlertAtManualCollectionEditor {
+		task ("Create a Collection via JSONWS") {
+			JSONAssetlist.addManualAssetListEntryWithItemType(
+				groupName = "Test Site Name",
+				itemSubtype = "All Subtypes",
+				itemType = "Web Content Article",
+				title = "Manual Collection Test");
+		}
+
+		task ("Navigate to Manual Collection Editor") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Manual Collection Test");
+		}
+
+		task ("Check if warning appears") {
+			Segmentation.checkSegmentationReEnableWarning();
+		}
+	}
+
+	@description = "LPS-151362: Given segmentation is disabled at Instance Settings
+	When user navigates to Segments page
+	Then the warning alert explains how to re-enable it"
+	@priority = "4"
+	test ViewReEnableAlertAtSegmentsList {
+		task ("Open the Segments List and check if warning appears") {
+			Segmentation.openSegmentsAdmin(siteURLKey = "guest");
+
+			Segmentation.checkSegmentationReEnableWarning();
+		}
+	}
+
+	@description = "LPS-151362: Given segmentation is disabled at Instance Settings
+	When user navigates to Experience simulation sidepanel
+	Then the warning alert explains how to re-enable it"
+	@priority = "3"
+	test ViewReEnabledAlertAtExperienceSimulationSidepanel {
+		task ("Add segment") {
+			Segmentation.openSegmentsAdmin(siteURLKey = "guest");
```

Comment:
Can you modify this block to use JSONWS? Like this example:
```
JSONSegmentsentry.addSegment(
	fieldName = "First Name",
	groupName = "Test Site Name",
	operator = "equals",
	segmentName = "First Segment",
	text = "Test");
```

Reply:
ok!!